src/components/molecules/Sidebar/Sidebar.tsx

```diff
@@ -72,6 +83,12 @@ const Sidebar = () => {
         >
           {hasUnreadMessages && <div className="notification"></div>}
           <span>Messages</span>
+          
+          {numberOfUnreadMessages > 0 && (
+                        <span>
+                          ({numberOfUnreadMessages}) <audio autoPlay ><source src={notificationSound} /></audio>
+                        </span>
+                      )}
```

Comment:
The spacing here looks incorrect, which shouldn't be possible if `prettier` has been run correctly. This is the same pattern of issue as I called out in my last review in https://github.com/sparkletown/sparkle/pull/1152#discussion_r574134034:

> - I noticed that the spacing/etc doesn't align to our standards. This should be automatically handled by `prettier` (and is enforced by our linters in CI, which is why this PR has an error status at the moment)
>   - It's interesting that you were even able to commit this, because prettier should get run automatically when you try and commit any code because of `husky` and `lint-staged`
>     - https://github.com/sparkletown/sparkle/blob/staging/package.json#L111-L123
>   - **Did you bypass commit hooks or anything like that by chance?** 
> 
> _Originally posted by @0xdevalias in https://github.com/sparkletown/sparkle/pull/1152#discussion_r574134034_

---

I also followed up on slack to try and get more information about if/why prettier wasn't automatically running for you when you tried to commit the code, as our git commit hooks should prevent that from even being possible.

---

This is the reason that you are currently getting the CI linting errors that will make this PR unmergeable in its current state:

- https://app.circleci.com/pipelines/github/sparkletown/sparkle/4827/workflows/5772541a-da2f-4c97-aa9a-198aa150b4c5/jobs/7940

![image](https://user-images.githubusercontent.com/753891/108024387-939f2900-7078-11eb-9ce9-d1265c813b67.png)